samwise.js

```diff
@@ -3,32 +3,32 @@
  *
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.